src/rpc/simulate/compressor/LrAmCompressor.h

```diff
@@ -276,24 +277,14 @@ namespace Rpc
       double setLambda();
       
       /**
-      * Get the IntraCorrelation (homopolymer) by reference.
+      * IntraCorrelation (homopolymer) object
       */
-      IntraCorrelation<D>& intraCorrelation();
-      
-      /**
-      * Pointer to the IntraCorrelation (homopolymer) class
-      */
-      IntraCorrelation<D>* intraCorrelationPtr_;
+      IntraCorrelation<D> intraCorrelation_;
```

Comment:
Is the intraCorrelation_ object private? (It's hard for me to tell if this section of code is public, protected or private in the review format, but please double check.  Objects marked with an underscore should be private or protected, and should private whenever possible. If you provide a public or protected member function to access a data variable, the variable (object) itself should be private.